### DISCUSSION TITLE
Show pit stop times

### DISCUSSION THREAD

**Discussion by ManCaveMedia** [f1multiviewer/issue-tracker#221] [Feature requests] [answered]
Since we have about all the car telemetry available in the app it would be nice to have the pit stop time added somewhere.

This could be part of the tire usage overlay:
![tires](https://user-images.githubusercontent.com/13760772/192218485-ff3126bb-440b-43df-b790-823c76062305.png)

Another option would be to give the pit stops their own overlay with in lap and out lap times and pit lane/pit stop delta, as far as that data is available.
Definitely a nice-to-have, not a need-to-have.

**Comment by nubbix** [comment]
Excellent suggestion

**Comment by jeppeolesen** [comment, marked as answer]
A variant of this was already added to MultiViewer, which uses the data we have available to summarise a stop. Unfortunately we don't have the stop time itself, but we can use the lap time data to calculate the total stop time.

![Screenshot 2025-03-22 at 17 46 36](https://github.com/user-attachments/assets/e03158c8-313f-4cd5-9135-96770edbcd63)
